tasks/libs/common/utils.py

```diff
@@ -125,18 +125,17 @@ def get_win_py_runtime_var(python_runtimes):
 
 
 def get_embedded_path(ctx):
-    embedded_path = ""
     base = os.path.dirname(os.path.abspath(__file__))
-    task_repo_root = os.path.abspath(os.path.join(base, ".."))
+    task_repo_root = os.path.abspath(os.path.join(base, "..", ".."))
     git_repo_root = get_root()
     gopath_root = f"{get_gopath(ctx)}/src/github.com/DataDog/datadog-agent"
 
     for root_candidate in [task_repo_root, git_repo_root, gopath_root]:
         test_embedded_path = os.path.join(root_candidate, "dev")
         if os.path.exists(test_embedded_path):
-            embedded_path = test_embedded_path
+            return test_embedded_path
 
-    return embedded_path
+    return ""
```

Comment:
```suggestion
    return None
```
or 
```suggestion
```

Since the only known caller seems to expect `None` if the path wasn't found:
https://github.com/DataDog/datadog-agent/blob/c3fc5e20617141fd0a0a5593d814fcda195698ac/tasks/libs/common/utils.py#L193-L196

Reply:
I did it that way to keep the previous behavior but I agree with you